src/wallet/crypter.h

```diff
@@ -42,16 +42,19 @@ class CMasterKey
     //! Use this for more parameters to key derivation (currently unused)
     std::vector<unsigned char> vchOtherDerivationParameters;
 
+    //! Default/minimum number of key derivation rounds
+    // 25000 rounds is just under 0.1 seconds on a 1.86 GHz Pentium M
+    // ie slightly lower than the lowest hardware we need bother supporting
+    static constexpr int DEFAULT_DERIVE_ITERATIONS = 25000;
```

Comment:
nit: this could be used in `src/wallet/test/wallet_crypto_tests.cpp` and `src/wallet/test/fuzz/crypter.cpp`